recipes/miniaudio/all/conanfile.py

```diff
@@ -12,47 +14,100 @@ class MiniaudioConan(ConanFile):
     license = ["Unlicense", "MIT-0"]
     settings = "os", "compiler", "build_type", "arch"
 
-    no_copy_source = True
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "header_only": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "header_only": True,
```

Comment:
```suggestion
        "header_only": False,
```

I would go for compiled library by default. C header only libs are evil and lead to ODR violations.

Reply:
That would change the expected behavior for current users, but fine by me.